pkg/networkmanager/interfaces.js

```diff
@@ -585,6 +585,18 @@ export function NetworkManagerModel() {
             };
         }
 
+        if (settings.wireguard) {
+            result.wireguard = {
+                listen_port: get("wireguard", "listen-port", 0),
+                peers: get("wireguard", "peers", []).map(peer => ({
+                    publicKey: peer['public-key'].v,
+                    endpoint: peer.endpoint?.v ?? "", // enpoint of a peer is optional
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19024-20230724-063723-aa0bc6c1-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)